# Velocity command configuration.
- topic_name: "/parrot_bebop_2/cmd_vel"
  ros_type_name: "geometry_msgs/msg/Twist"
  gz_type_name: "gz.msgs.Twist"
  lazy: true
  direction: ROS_TO_GZ

# Motor speed command configuration.
- topic_name: "/parrot_bebop_2/gazebo/command/motor_speed"
  ros_type_name: "std_msgs/msg/Float32"
  gz_type_name: "gz.msgs.Float"
  lazy: true
  direction: ROS_TO_GZ

# Enable Quadrotor command configuration.
- topic_name: "/parrot_bebop_2/enable"
  ros_type_name: "std_msgs/msg/Bool"
  gz_type_name: "gz.msgs.Boolean"
  lazy: true
  direction: ROS_TO_GZ

# Odometry state configuration.
- topic_name: "/model/parrot_bebop_2/odometry"
  ros_type_name: "nav_msgs/msg/Odometry"
  gz_type_name: "gz.msgs.Odometry"
  lazy: true
  direction: GZ_TO_ROS

# Pose state configuration.
- topic_name: "/model/parrot_bebop_2/pose"
  ros_type_name: "geometry_msgs/msg/Pose"
  gz_type_name: "gz.msgs.Pose"
  lazy: true
  direction: GZ_TO_ROS

# Initial Pose configuration.
- topic_name: "/parrot_bebop_2/set_pose"
  ros_type_name: "geometry_msgs/msg/Pose"
  gz_type_name: "gz.msgs.Pose"
  lazy: true
  direction: ROS_TO_GZ

# Simulation Clock 
- topic_name: "/clock"
  ros_type_name: "rosgraph_msgs/msg/Clock"
  gz_type_name: "gz.msgs.Clock"
  lazy: true
  direction: GZ_TO_ROS

# rgbd_camera_bridge configuration.


- topic_name: "/camera/image"
  ros_type_name: "sensor_msgs/msg/Image"
  gz_type_name: "gz.msgs.Image"
  lazy: true
  direction: GZ_TO_ROS

- topic_name: "/camera/camera_info"
  ros_type_name: "sensor_msgs/msg/CameraInfo"
  gz_type_name: "gz.msgs.CameraInfo"
  lazy: true
  direction: GZ_TO_ROS
